File: api/code-samples/python/admin/setSpamThrottlerWhiteList.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = ["0xfdeedbb2fe5b48d5b49e435ba00e0358740d0cf5"]

sdk = OpenSDK(host)
admin_response = sdk.admin.set_spam_throttler_white_list(address)

print(json.loads(admin_response.response.data))

File: web3rpc/api/code-samples/javascript/admin/setSpamThrottlerWhiteList.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const address = ['0xfdeedbb2fe5b48d5b49e435ba00e0358740d0cf5']
    sdk.admin.setSpamThrottlerWhiteList(address, {}, (err, data, response) => {
        console.log(data);
    });
}
)()